workspaces/kiali/plugins/kiali-backend/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@backstage-community/plugin-kiali-backend",
+  "version": "1.19.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "backend-plugin",
+    "supported-versions": "1.31.3",
+    "pluginId": "kiali",
+    "pluginPackages": [
+      "@backstage-community/plugin-kiali",
+      "@backstage-community/plugin-kiali-backend"
+    ]
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "tsc": "tsc",
+    "prettier:check": "prettier --ignore-unknown --check .",
+    "prettier:fix": "prettier --ignore-unknown --write .",
+    "lint:check": "backstage-cli package lint",
+    "lint:fix": "backstage-cli package lint --fix",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "postversion": "yarn run export-dynamic",
+    "export-dynamic": "janus-cli package export-dynamic-plugin",
+    "export-dynamic:clean": "janus-cli package export-dynamic-plugin --clean"
+  },
+  "configSchema": "config.d.ts",
+  "dependencies": {
+    "@backstage/backend-defaults": "^0.5.0",
+    "@backstage/backend-plugin-api": "^1.0.0",
+    "@backstage/catalog-client": "^1.7.0",
+    "@backstage/catalog-model": "^1.7.0",
+    "@backstage/config": "^1.2.0",
+    "@backstage/errors": "^1.2.4",
+    "@backstage/plugin-auth-node": "^0.5.2",
+    "axios": "^1.7.4",
+    "express": "^4.18.2",
+    "moment": "^2.29.4",
+    "yn": "^4.0.0"
+  },
+  "devDependencies": {
+    "@backstage/backend-test-utils": "1.0.0",
+    "@backstage/cli": "0.27.1",
+    "@janus-idp/cli": "1.16.0",
```

Comment:
Does the `@janus-idp/cli`  devDependency need to be removed @schultzp2020 ?

Reply:
I think that we need to keep it to support Janus like other plugins

Check
[feedback plugin](https://github.com/backstage/community-plugins/blob/730359c215a1cff38339e8e26c0451e6a781d864/workspaces/feedback/plugins/feedback/package.json#L56)
[redhat-resource-optimization]( https://github.com/backstage/community-plugins/blob/730359c215a1cff38339e8e26c0451e6a781d864/workspaces/redhat-resource-optimization/package.json#L49)

Reply:
We plan on removing the janus cli dependencies from other plugins as well. We can leave here for now and tackle that in a separate PR when we remove it from all plugins.